libs/libvtrcapnproto/README.md

```diff
@@ -0,0 +1,84 @@
+Capnproto usage in VTR
```

Comment:
This is a great overview of the tool, how it fits together, and how to extend it in the future. Thanks!